config/config.go

```diff
@@ -165,13 +146,13 @@ func (l *Loader) Environment() string {
 	return _devEnv
 }
 
-// Path returns path to the yaml configurations
-func (l *Loader) Path() string {
+// Paths returns paths to the yaml configurations
+func (l *Loader) Paths() []string {
 	if path, ok := l.lookUp(l.EnvironmentPrefix() + _configDir); ok {
-		return path
+		return []string{path}
```

Comment:
If ennvironment variable is set, then never, but by default it is multiple strings: we look in `.` and `./config`